### ISSUE TITLE
Thousand delimited scores

### ISSUE THREAD

**Issue by PedersenThomas** [hoovercj/beatsaber-leaderboard-react#8]
On the page for a given song the score is as 368115 but I think it would be easier to read with some delimiters like: 368,115

**Comment by hoovercj** [comment]
Good idea. I've marked it as `help wanted` and `good first issue` if you want to help :-)
